src/avram/queryable.cr

```diff
@@ -221,7 +221,7 @@ module Avram::Queryable(T)
     cache_store.fetch(cache_key(:any?), as: Bool) do
       queryable = clone
       new_query = queryable.query.limit(1).select("1 AS one")
-      results = database.query_one?(new_query.statement, args: new_query.args, queryable: schema_class.name, as: Int32)
+      results = database.query_one?(new_query.statement, args: new_query.args, queryable: schema_class.name, as: (Int32 | Int64))
```

Comment:
I think we can just use `as: Int64` here. The actual return value is irrelevant. We only really care if there is a value.

Reply:
because of other databases this needs to be Int32 and Int64.